src/material/select/select.ts

```diff
@@ -426,6 +426,9 @@ export class MatSelect extends _MatSelectMixinBase implements AfterContentInit,
   /** Object used to control when error messages are shown. */
   @Input() errorStateMatcher: ErrorStateMatcher;
 
+  /** Time to wait in milliseconds after the last keystroke before moving focus to an item. */
+  @Input() letterKeyDebounceInterval: number;
```

Comment:
As a public API, "letterKey" seems a little too much like an implementation detail. How about `typeaheadDebounceInterval`?

Reply:
Sure, that sounds better. Changed.